qubes-rpc/qubes-sync-clock

```diff
@@ -25,12 +25,12 @@ import subprocess
 def main():
     stdin = sys.stdin.read(25)
 
-    date_out = stdin.strip()
+    untrusted_date_out = stdin.strip()
 
-    if not re.match(r'^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+00:?00$', date_out):
+    if not re.match(r'^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+00:?00$', untrusted_date_out):
         sys.stderr.write('Invalid date received, aborting!')
         sys.exit(1)
-    subprocess.check_call(['date', '-u', '-Iseconds', '-s', date_out],
+    subprocess.check_call(['date', '-u', '-Iseconds', '-s', untrusted_date_out],
```

Comment:
After validation of the value, drop `untrusted_` prefix, see here: https://www.qubes-os.org/doc/coding-style/#security-coding-guidelines